### DISCUSSION TITLE
Upload Multiple SBOMs in single project

### DISCUSSION THREAD

**Discussion by sahil3112** [DependencyTrack/dependency-track#1367] [Q&A] [answered]
Hello Everyone,
I have a couple of projects and each project has several SBOMs so is there any way to upload more than one SBOM in a single project and keep track using SBOMs name (like a project with some sub-projects) so that I can manage Vulnerability more easily 

Thanks

**Comment by stevespringett** [comment, marked as answer]
Subprojects are not currently supported, but are on the roadmap. Most orgs use the CycloneDX CLI to merge multiple BOMs resulting in a single BOM for a project? It's a very common use case, especially when the backend services are written in Java, Go, etc, and the frontend is JavaScript.

https://github.com/CycloneDX/cyclonedx-cli

**Reply by sahil3112** [reply]
Thanks @stevespringett
